specification/dartLangSpec.tex

```diff
@@ -41,6 +41,11 @@
 % version of the language which will actually be specified by the next stable
 % release of this document.
 %
+% Sep 2025
+% - Remove the rules about scoping with DartDoc comments. This topic is now
+%   handled by each of the tools that process those comments in any way, and
+%   this document only specifies the syntax of DartDoc comments.
```

Comment:
Why specify the syntax of DartDoc comments?

To the language, they're just comments.